src/org/jetbrains/plugins/scala/compiler/ScalaHydraCompilerConfigurationPanel.scala

```diff
@@ -37,20 +40,53 @@ class ScalaHydraCompilerConfigurationPanel(project: Project, settings: HydraComp
   userTextField.getDocument.addDocumentListener(documentAdapter)
   passwordTextField.getDocument.addDocumentListener(documentAdapter)
   passwordTextField.addFocusListener(focusListener)
-  hydraVersionComboBox.setItems(Versions.loadHydraVersions)
+  hydraVersionComboBox.setItems(downloadHydraVersions)
   downloadButton.addActionListener((_: ActionEvent) => onDownload())
+  noOfCoresComboBox.setItems(Array.range(1, Runtime.getRuntime.availableProcessors()/2 + 1).map(_.toString))
+  sourcePartitionerComboBox.setItems(SourcePartitioner.values.map(_.value).toArray)
+
+  hydraStoreDirectoryField.addBrowseFolderListener("", "Hydra Store Path", null, fileChooserDescriptor, TextComponentAccessor.TEXT_FIELD_WHOLE_TEXT)
```

Comment:
Is there a way to provide a description for what each option is for? If yes, please use the description we have in the documentation: https://docs.triplequote.com/user-guide/#configuration